### PR TITLE
update aws

### PR DESCRIPTION
**Pull request by maryamsulemani97**
This PR updates `learn/cookbooks/aws.md`

part of https://github.com/meilisearch/documentation/issues/1719

### CODE REVIEW

File: learn/cookbooks/aws.md [hunk 1]
```diff
@@ -36,72 +36,72 @@ Select the specifications of the server you want Meilisearch to run on.
 We recommend prioritizing memory allocation for better Meilisearch performance.
 
 ::: tip
-The **free tier** is sufficient for tests or prototypes, but not recommended for large datasets.
+The "free tier" is sufficient for tests or prototypes, but not recommended for large datasets.
```
Comment (guimachiavelli) [inline learn/cookbooks/aws.md:39]: How about we leave it without quotes? This way we are referring to whatever free tier Amazon has and don't need to bother with its proper name.

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
Added a minor comment. If you think it's a bad idea, you mark it as resolved and re-request the review ☺️

File: learn/cookbooks/aws.md (hunk 1 above)
Comment (maryamsulemani97) [inline learn/cookbooks/aws.md:39]:
```suggestion
The free tier is sufficient for tests or prototypes, but not recommended for large datasets.
```

**Review by guimachiavelli — APPROVED** [review]
😶‍🌫️

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**